lib/rules/aria-roledescription.json

```diff
@@ -0,0 +1,13 @@
+{
+	"id": "aria-roledescription",
+	"selector": "[aria-roledescription]",
+	"excludeHidden": false,
```

Comment:
This is different from `aria-allowed-attr`, why did you change this? I don't think this is likely to cause false positives, but probably not any less than `aria-allowed-attr`. IMO it's probably safer to not do this and rely on people testing axe in multiple page states instead.